src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -50,11 +46,34 @@ public CommandResult execute() {
         final List<ReadOnlyPerson> matchedPersons = new ArrayList<>();
         for (ReadOnlyPerson person : addressBook.getAllPersons()) {
             final Set<String> wordsInName = new HashSet<>(person.getName().getWordsInName());
-            if (!Collections.disjoint(wordsInName, keywords)) {
+            if (!disjointIgnoreCase(wordsInName, keywords)) {
                 matchedPersons.add(person);
             }
         }
         return matchedPersons;
     }
+    /**
+     * This is a case insensitive version of disjoint method
+     *
+     * @param coll1
+     * @param coll2
```

Comment:
as mentioned during the lecture, such tags with no further explanation or variable name is self explanatory, can exclude them from the javadocs.